plugins/woocommerce-admin/client/products/tour/block-editor/block-editor-tour.tsx

```diff
@@ -0,0 +1,211 @@
+/**
+ * External dependencies
+ */
+import { TourKit } from '@woocommerce/components';
+import { Guide } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+
+import './style.scss';
+
+interface Props {
+	isTourOpen: boolean;
+	dismissModal: () => void;
+	openGuide: () => void;
+	closeGuide: () => void;
+	isGuideOpen: boolean;
+}
+
+const BlockEditorTour = ( {
+	isTourOpen,
+	dismissModal,
+	openGuide,
+	isGuideOpen,
+	closeGuide,
+}: Props ) => {
+	useEffect( () => {
+		if ( isTourOpen ) {
+			recordEvent( 'block_product_editor_spotlight_view' );
+		}
+	}, [ isTourOpen ] );
+
+	if ( isTourOpen ) {
+		return (
+			<TourKit
+				config={ {
+					steps: [
+						{
+							meta: {
+								name: 'woocommerce-block-editor-tour',
+								primaryButton: {
+									text: __(
+										'View highlights',
+										'woocommerce'
+									),
+								},
+								descriptions: {
+									desktop: __(
+										"We designed a brand new product editing experience to let you focus on what's important.",
+										'woocommerce'
+									),
+								},
+								heading: __(
+									'Meet a streamlined product form',
+									'woocommerce'
+								),
+							},
+							referenceElements: {
+								desktop: '#adminmenuback',
+							},
+						},
+					],
+					closeHandler: ( _steps, _currentStepIndex, source ) => {
+						dismissModal();
+						if ( source === 'done-btn' ) {
+							recordEvent(
+								'block_product_editor_spotlight_view_highlights'
+							);
+							openGuide();
+						} else {
+							recordEvent(
+								'block_product_editor_spotlight_dismissed'
+							);
+						}
+					},
+					options: {
+						effects: {
+							arrowIndicator: false,
+							overlay: false,
+							liveResize: {
+								rootElementSelector: '#adminmenuback',
+								resize: true,
+							},
+						},
+						portalParentElement:
+							document.getElementById( 'wpbody' ),
+						popperModifiers: [
+							{
+								name: 'bottom-left',
+								enabled: true,
+								phase: 'beforeWrite',
+								requires: [ 'computeStyles' ],
+								fn: ( { state } ) => {
+									state.styles.popper.top = 'auto';
+									state.styles.popper.left = 'auto';
+									state.styles.popper.bottom = '10px';
+									state.styles.popper.transform =
+										'translate3d(10px, 0px, 0px)';
+								},
+							},
+						],
+					},
+				} }
+			/>
+		);
+	} else if ( isGuideOpen ) {
+		return (
+			<Guide
```

Comment:
Could we move this into it's own component - `block-editor-guide`? and use it within the block editor tour wrapper?